### PR TITLE
Delete account

### PR DESCRIPTION
**Pull request by mattgle**
## Summary

- Delete account
- Move Settings options

## Screenshots

https://user-images.githubusercontent.com/44204622/197062748-8ce9cc40-5a57-4780-8dd5-6f26192cbd10.mp4

## Github Issue

https://github.com/Psychedelic/plug-mobile/issues/509

### CODE REVIEW

File: src/redux/slices/keyring.ts [hunk 1]
```diff
@@ -296,6 +296,38 @@ export const createSubaccount = createAsyncThunk(
...
+        dispatch(
+          setCurrentPrincipal({
+            icpPrice: state.icp.icpPrice,
+            walletId: mainWalletId,
+          })
+        );
+      }
+      return { walletId };
+    } catch (e: any) {
+      return rejectWithValue({ error: e.message });
```
Comment (ignaciosantise) [inline src/redux/slices/keyring.ts:326]:
```suggestion
      return rejectWithValue(e.message);
```

File: src/screens/tabs/Profile/modals/Accounts/index.tsx [hunk 2]
```diff
@@ -100,6 +136,15 @@ const Accounts = ({ modalRef, onClose, ...props }) => {
       });
     }
 
+    if (isImportedAccount) {
+      options.push({
+        id: isSelectedAccount ? 4 : 3,
```
Comment (ignaciosantise) [inline src/screens/tabs/Profile/modals/Accounts/index.tsx:141]: we could use `options.length + 1` as ID

File: yarn.lock [hunk 3]
```diff
@@ -1777,17 +1777,31 @@
     cross-fetch "^3.1.4"
     crypto-js "^4.1.1"
 
-"@psychedelic/plug-controller@0.23.4":
-  version "0.23.4"
-  resolved "https://npm.pkg.github.com/download/@Psychedelic/plug-controller/0.23.4/92ba5923065d37ea322cfc758861da0694959cfa#92ba5923065d37ea322cfc758861da0694959cfa"
-  integrity sha512-fwdlVPZidcHFt6x+o5OXKSyJi/VA7htkpZJ6UcU6WromEsBenaHKuRPdfJgJzRpY5AMWckgLKf2HVPL1fHxxJg==
+"@psychedelic/dab-js@1.5.0-beta.1":
```
Comment (ignaciosantise) [inline yarn.lock:1780]: dab-js beta?

**Comment by ignaciosantise** [comment]
@mattgle did you test in ios?

File: yarn.lock (hunk 3 above)
Comment (mattgle) [reply yarn.lock:1780]: That's from the controllers version

Comment (ignaciosantise) [reply yarn.lock:1780]: yep, can you check if its ok to use that? why not a release?

Comment (mattgle) [reply yarn.lock:1780]: wil do

**Review by ignaciosantise — APPROVED** [review]
🚀

**Pull request merged**